extensions/ql-vscode/src/data-extensions-editor/queries/csharp.ts

```diff
@@ -0,0 +1,198 @@
+import { Query } from "./query";
+
+export const fetchExternalApisQuery: Query = {
+  mainQuery: `/**
+ * @name Usage of APIs coming from external libraries
+ * @description A list of 3rd party APIs used in the codebase.
+ * @tags telemetry
+ * @id csharp/telemetry/fetch-external-apis
```

Comment:
It seems like the `id` used in C# queries is `cs` (e.g. [here](https://github.com/github/codeql/blob/main/csharp/ql/src/Telemetry/SupportedExternalApis.ql)). It's probably best to already set this to `cs` to ensure we don't need to update this when upstreaming it.